docs/maui/views/popup-service.md

```diff
@@ -166,7 +191,7 @@ Expanding on the previous example the following implementation can be added to t
 [RelayCommand]
 void OnCancel()
 {
-    popupService.ClosePopup();
+    popupService.ClosePopupAsync(Shell.Current);
 }
 ```
```

Comment:
This needs to be an async method and we need to `await` `ClosePopupAsync`.

For simplicity, let's use `async`/`await` instead of returning a `Task`, which is also valid here.

```cs
[RelayCommand]
async Task OnCancel()
{
    await popupService.ClosePopupAsync(Shell.Current);
}
```